src/software/jetson_nano/services/motor.cpp

```diff
@@ -593,6 +595,11 @@ TbotsProto::MotorStatus MotorService::poll(const TbotsProto::MotorControl& motor
     return motor_status;
 }
 
+void MotorService::setTargetVelocity(uint8_t chip_select, int target)
```

Comment:
We aren't commanding velocities anymore, so I suggest removing this function

Reply:
done